networkx/algorithms/tree/tests/test_branchings.py

```diff
@@ -199,6 +200,26 @@ def test_greedy_max1():
     assert_equal_branchings(B, B_)
 
 
+def test_greedy_branch_kind():
+    G = G1()
+    with pytest.raises(nx.NetworkXException, match="Unknown value for `kind`."):
+        B = branchings.greedy_branching(G, kind="lol")
+
+
+def test_edges():
```

Comment:
Is this a fair description of the test?
```suggestion
def test_greedy_branching_for_unsortable_nodes():
```